maps-utils-ktx/src/main/java/com/google/maps/android/ktx/LatLng.kt

```diff
@@ -15,12 +15,65 @@
  *
  */
 @file:Suppress("NOTHING_TO_INLINE")
+
 package com.google.maps.android.ktx
 
 import com.google.android.gms.maps.model.LatLng
 import com.google.maps.android.PolyUtil
 import com.google.maps.android.SphericalUtil
 
+/**
+ * Computes where the given [latLng] is contained on or near this Polyline within a specified
+ * tolerance in meters.
+ *
+ * @param latLng the LatLng to inspect
+ * @param geodesic if this polyline is geodesic or not
+ * @param tolerance the tolerance in meters
+ * @return true if [latLng] is on this path, otherwise, false
+ *
+ * @see PolyUtil.isLocationOnPath
+ */
+inline fun List<LatLng>.isLocationOnPath(
+    latLng: LatLng,
+    geodesic: Boolean,
+    tolerance: Double = 0.1
+): Boolean = PolyUtil.isLocationOnPath(latLng, this, geodesic, tolerance)
+
+/**
+ * Checks whether or not [latLng] lies on or is near the edge of this polygon within a tolerance
+ * (in meters) of [tolerance]. The default value is [PolyUtil.DEFAULT_TOLERANCE].
+ *
+ * @param latLng the LatLng to inspect
+ * @param geodesic if this polygon is geodesic or not
+ * @param tolerance the tolerance in meters
+ * @return true if [latLng] lies on or is near the edge of this Polygon, otherwise, false
+ *
+ * @see PolyUtil.isLocationOnEdge
+ */
+inline fun List<LatLng>.isOnEdge(
+    latLng: LatLng,
+    geodesic: Boolean,
+    tolerance: Double = 0.1
```

Comment:
`0.1` should be `DEFAULT_TOLERANCE` from the Java lib

Reply:
That's what I meant to do now that AMU 1.0.1 is live on maven. I'll need to bump the version used by the KTX lib and update this.

Reply:
Actually I'll address this as a separate PR (with version bumping AMU to 1.0.1). Will update the PR with doc edits though.